gleam/README.md

```diff
@@ -0,0 +1,20 @@
+# Gilded Rose starting position in Gleam
+
+I assume you have [installed](https://gleam.run/getting-started/installing/) `gleam` on your system.
+
+## Run unit tests from the command line
+
+```sh
+$ gleam test
+```
+
+## Get the TextTest fixture on the command line
+
+Execute it on the command line with an argument for the number of days:
+
+```sh
+$ gleam run -- --days=30
+```
+
+Compare the output to [stdout.gr](../texttests/ThirtyDays/stdout.gr) for validation.
+There are instructions in the [TextTest Readme](../texttests/README.md) for setting up TextTest. 
```

Comment:
I could instead replace this with the

```
$ gleam run > /tmp/output.txt
$ diff /tmp/output.txt ../texttests/ThirtyDays/stdout.gr
```

If preferred.

Reply:
It's ok as provi